object_detection/src/application/script.py

```diff
@@ -88,3 +88,73 @@ def process(self):
 
 
         return direction
+
+
+class Detection:
+    MODEL_PATH = "../../object_detection_model/yolov5s.pt"
+    img_array = []
+    current_direction = ''
+
+    def __init__(self) -> None:
+        self.model = torch.hub.load('ultralytics/yolov5', 'custom', self.MODEL_PATH)
+        
+        self.target_coords = (None, None) # Initialize target coordinates to None
```

Comment:
@Toniodujinrin Victor appears to be doing this here.
CC: @ThePyLord

Reply:
oh, I see ... Thanks